### ISSUE TITLE
Is evaluation server still alive? My submission is stuck at "Running" for hours

### ISSUE THREAD

**Issue by nguyendinhson-kaist** [DeepSportradar/instance-segmentation-challenge#6]

**Comment by mistasse** [comment]
Investigating and optimizing the OM metric computation. ;-)

**Comment by nguyendinhson-kaist** [comment]
Thank you for letting me know that. If possible, please let me know when the server works

**Comment by mistasse** [comment]
The server never really stopped working, I suspect that the worker crashes when there are too many masks in the prediction, but this is something that is difficult to optimize because I chose to bind within MMDet's metric computation which has different control flow from what I would need to compute OM efficiently.

It is extremely difficult to progress on this issue as the EvalAI containers cannot easily be inspected and are not responsive at all given the environment setup. All my attempts have only broken the evaluation. I will still be trying something over the week-end, but might end up running the script locally on your results if I can't get it to work, as I did for some submissions of the first phase.

I will keep you tuned ;-) If this number of predictions is really the problem, I might as well just consider the top 200 most confident masks, having more doesn't seem promising anyway, but this would be sad because I wanted to let people try to hack the metric in whatever way they could imagine.
